### PR TITLE
User story 20 sound effects

### PR DESCRIPTION
**Pull request by Figer-op**
add description here ... jag är klkar

### CODE REVIEW

File: DAT257AgileProject/Assets/Scenes/Game/First World.unity
Comment (Caramajau) [inline DAT257AgileProject/Assets/Scenes/Game/First World.unity:1]: Why change?

File: DAT257AgileProject/Assets/Resources/ScriptableObjects/Audio/Sound/ExclamationMarkAppears.asset
Comment (Caramajau) [inline DAT257AgileProject/Assets/Resources/ScriptableObjects/Audio/Sound/ExclamationMarkAppears.asset:1]: Should maybe be a little quieter?

File: DAT257AgileProject/Assets/Resources/ScriptableObjects/Audio/Sound/StartFishSound.asset
Comment (Caramajau) [inline DAT257AgileProject/Assets/Resources/ScriptableObjects/Audio/Sound/StartFishSound.asset:1]: Should maybe be a little quieter?

**Review by Caramajau — CHANGES_REQUESTED** [review]
Lastly you should probably update the credits, though you might want to wait with that since User Story 32 and User Story 29 has done things in the main menu. After that I believe no other User Story should affect the main menu, but could be wrong.

**Review by Caramajau — APPROVED** [review]

**Review by Oskmey — APPROVED** [review]
LGTM

**Pull request merged**